model/directory_status.py

```diff
@@ -24,8 +46,15 @@ def status(self):
 
     @status.setter
     def status(self, status):
+        if status not in DirectoryStatus.VALID_STATUS_LIST:
+            raise Exception("Invalid status written to DirectoryStatus object.")
         self._status = status
 
+    def status_equals(self, status):
```

Comment:
Method comments for both this and `status` above please.

Reply:
Added comments for all 3 functions.